hack/api-docs/mkdocs.yml

```diff
@@ -62,6 +62,7 @@ nav:
       - Password: api/generator/password.md
       - Fake: api/generator/fake.md
       - Webhook: api/generator/webhook.md
+      - Github: api/generator/github.md
```

Comment:
I believe you also need to add it to the index in `api/generator/index.md'

Reply:
After checking `api/generator/index.md` and other generators like `password.md`  ref to generator md doc only present in `hack/api-docs/mkdocs.yml`